.github/workflows/build_config_server.yaml

```diff
@@ -20,8 +20,8 @@ permissions:
   contents: 'read'
 
 env:
-  IMAGE_NAME: config-server
-  REGISTRY: ${{ github.event_name == 'pull_request' && vars.DOCKER_REGISTRY || 'steeltoeoss' }}
+  IMAGE_NAME: ${{ github.event_name == 'pull_request' && 'config-server-pr' || 'config-server' }}
```

Comment:
Can this include the PR number, so that multiple PRs don't overwrite each other?
https://stackoverflow.com/questions/60771258/how-to-get-the-pull-request-number-from-predefined-variable-in-azure-build-pipel

Reply:
There are likely a variety of strategies we could use here, but I don't think we've ever had (or I can forecast a reason for) multiple simultaneous PRs for the same image... _If_ we do need that support, I'd be more inclined to put the PR number into a tag than the image name as we'll get massive sprawl in the container repository that's harder to clean up... Here are some screenshots of what the changes already in here look like in the registry:

![image](https://github.com/user-attachments/assets/493c7dfd-a5f9-404f-b3d5-867062b692ee)
- That is the list we'd want to clean up after every few PRs

![image](https://github.com/user-attachments/assets/0a601423-38e8-447c-9126-ae4e0f7c76c0)
- Putting the PR number in tags would end up expanding this list instead, which can be cleaned up with some automation


But given how infrequently we update these images, I'm not sure it's worth any additional effort

Reply:
Thanks